lib/managers/files.js

```diff
@@ -541,6 +542,26 @@ Files.prototype.uploadFile = function(parentFolderID, filename, content, callbac
 };
 
 /**
+ * Uploads a new file. Unlike non-upload methods, this method will not perform any retries.
+ * This method currently does not support any optional parameters such as contentModifiedAt.
+ *
+ * API Endpoint: '/files/content'
+ * Method: POST
+ *
+ * @param {string} parentFolderID - the id of the parent folder to upload to
+ * @param {string} filename - the file name that the uploaded file should have
+ * @param {string} path - the full path of the file that needs to be uploaded. The path includes the filename
+ * @param {Function} callback - called with data about the upload if successful, or an error if the
+ * upload failed
+ * @returns {void}
+ */
+Files.prototype.uploadFileFromPath = function(parentFolderID, filename, path, callback) {
+
+	var stream = fs.createReadStream(path);
```

Comment:
You should handle any errors thrown here!  This method is supposed to be asynchronous, so any errors produced should be passed to the callback rather than thrown.